### ISSUE TITLE
Input being required should not result in aria-invalid="true"

### ISSUE THREAD

**Issue by micamp** [angular/components#18140]
#### Bug, feature request, or proposal:
Input being required should not result in aria-invalid="true".

#### What is the expected behavior?
An input field that is required should be marked as required in the HTML (which it is), but should not be marked as aria-invalid until the user enters an invalid value.

#### What is the current behavior?
An input field that is required is marked as both required and aria-invalid before any content is entered by the user.

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>
1. Open [the AM input example page](https://material.angular.io/components/input/examples)
1. Find the "Input with a custom ErrorStateMatcher" example
1. Inspect the HTML of the input

Expected: The input element should have aria-invalid="false" or no aria-invalid attribute.
Actual: The input element has aria-invalid="true".

#### What is the use-case or motivation for changing an existing behavior?
It is confusing to users with a screen reader why it would be invalid before they have even entered any content.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Latest. It is on your example page.

#### Is there anything else we should know?
Thank you for your attention on this issue.

**Comment by IAfanasov** [comment]
Can not reproduce it on the current version 9.2.1. The `aria-invalid` is set to false. Screen record: 

![aria-invalid-false](https://user-images.githubusercontent.com/10142653/80469509-f0422380-8940-11ea-88e4-deee7de68189.gif)

Tested on Mac, browsers Chrome, Brave and Firefox
